clang/utils/TableGen/ClangAttrEmitter.cpp

```diff
@@ -3653,7 +3653,8 @@ static void GenerateAppertainsTo(const Record &Attr, raw_ostream &OS) {
 static void GenerateMutualExclusionsChecks(const Record &Attr,
                                            const RecordKeeper &Records,
                                            raw_ostream &OS,
-                                           raw_ostream &MergeOS) {
+                                           raw_ostream &MergeDeclOS,
+                                           raw_ostream &MergeStmtOS) {
```

Comment:
I'm also not in love with this function generating code into three separate streams, but I couldn't think of a better approach that avoids walking the entire list of records from Attr.td multiple times (which slows down compile speeds because there are a fair amount of records in that file these days).